### PR TITLE
Feature/getting started

### PR DESCRIPTION
**Pull request by jesseconductor**

### CODE REVIEW

File: docs/getting_started/job_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Managing Your Jobs
+
+Jobs can be managed via the (web dashboard | https://dashboard.conductortech.com)
+
+**SCREENSHOT OF CONDUCTOR DASHBOARD**
```
Comment (kaylegraham) [inline docs/getting_started/job_monitoring.md]: is the image supposed to be here already or is the remaining placeholder intentional?

File: docs/getting_started/downloading_files.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#Downloading Files
+
+When a task has been completed, the output data from that job is made available to download. Files can be downloaded by using the Desktop Companion or the Conductor command-line interface (CLI).
```
Comment (kaylegraham) [inline docs/getting_started/downloading_files.md]: We should probably be more consistent with how we talk about the app. On the marcom site, and our outgoing communications, I've been referring to it as Conductor Companion, Companion, or using lowercase when describing as a companion app/ desktop companion app

**Review by kaylegraham — APPROVED** [review]

File: docs/getting_started/job_monitoring.md (hunk 1 above)
Comment (jesseconductor) [reply docs/getting_started/job_monitoring.md]: Thanks for catching that. I did put the screenshot in but forgot to remove that.

File: docs/getting_started/downloading_files.md (hunk 2 above)
Comment (jesseconductor) [reply docs/getting_started/downloading_files.md]: I'll update all the references to say Conductor Companion

**Pull request merged**